app/v1/services/DeleteCustomEmploymentService.scala

```diff
@@ -48,4 +48,17 @@ class DeleteCustomEmploymentService @Inject() (connector: DeleteCustomEmployment
       "SERVICE_UNAVAILABLE"       -> InternalError
     )
 
+    val hipErrors = Map(
+      "1117" -> TaxYearFormatError,
+      "1215" -> NinoFormatError,
+      "1217" -> EmploymentIdFormatError,
+      "5000" -> RuleTaxYearNotSupportedError,
+      "5010" -> NotFoundError,
+      "1222" -> RuleDeleteForbiddenError,
+      "4200" -> RuleOutsideAmendmentWindowError
```

Comment:
RULE_OUTSIDE_AMENDMENT_WINDOW only applied to V2. Can it be removed from v1 code and tests